### PR TITLE
Updating Trunk Seats

### PR DESCRIPTION
**Pull request by reustle**
The seats at the big tables (where they prefer the laptops) are hard wooden chairs, and are particularly uncomfortable after an hour. I'm tempted to even give them a 2...

### CODE REVIEW

**Comment by pqvst** [comment]
I'd suggest downgrading `comfort` to 3 in that case. I still find the vacancy of seats to be pretty good there so I think leaving `seats` at 4 is probably fine? (I'm going to rename the seats field to vacancy)

File: data/cities/tokyo.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     "station": "JY20,G01,Z01,F16,C03,F15,JY19",
     "wifi": 5,
     "power": 5,
-    "seats": 4,
+    "seats": 3,
```
Comment (reustle) [inline data/cities/tokyo.json]:
```suggestion
    "seats": 4,
```

File: data/cities/tokyo.json [hunk 2]
```diff
@@ -150,7 +150,7 @@
...
     "comfort": 5,
```
Comment (reustle) [inline data/cities/tokyo.json]:
```suggestion
    "comfort": 3,
```

File: data/cities/tokyo.json [hunk 3]
```diff
@@ -151,7 +151,7 @@
     "wifi": 5,
     "power": 5,
     "seats": 4,
-    "comfort": 5,
+     "comfort": 3,
```
Comment (reustle) [inline data/cities/tokyo.json]:
```suggestion
    "comfort": 3,
```

**Comment by reustle** [comment]
Updated

**Pull request merged**